server/services/participants.js

```diff
@@ -127,6 +127,14 @@ const getParticipantByID = async (participantInfo) => {
   return participant;
 };
 
+const getParticipantCurrentStatus = async (participantId) => {
+  const participant = await dbClient.db[collections.PARTICIPANTS_STATUS].findOne({
```

Comment:
Does this take into account that participants can have more than one current status as a time? Does it matter?